### PR TITLE
Accept rhel edge installer as image type

### PR DESCRIPTION
**Pull request by biancarosa**

### CODE REVIEW

File: pkg/models/images.go [hunk 1]
```diff
@@ -35,8 +38,8 @@ func (i *Image) ValidateRequest() error {
 	if i.Commit == nil || i.Commit.Arch == "" {
 		return errors.New(ArchitectureCantBeEmptyMessage)
 	}
-	if i.OutputType != "tar" {
-		return errors.New(OnlyTarAcceptedMessage)
+	if i.ImageType != ImageTypeCommit && i.ImageType != ImageTypeInstaller {
```
Comment (omaciel) [inline pkg/models/images.go:41]: Should this be an `OR`?

**Review by omaciel — COMMENTED** [review]
Left a tiny comment :)

File: pkg/models/images.go (hunk 1 above)
Comment (omaciel) [reply pkg/models/images.go:41]: Nevermind. :/

**Comment by omaciel** [comment]
LGTM

**Comment by chambridge** [comment]
@biancarosa is this ready for review? Just asking as its in Draft still. Otherwise it looks fine and I'll happily :+1:.

**Comment by biancarosa** [comment]
@chambridge just marked as ready!

**Review by chambridge — APPROVED** [review]
👍

**Pull request merged**